consensus/blake3pow/poem.go

```diff
@@ -29,15 +29,23 @@ func (blake3pow *Blake3pow) CalcOrder(header *types.Header) (*big.Int, int, erro
 	// PRIME
 	// PrimeEntropyThreshold number of zone blocks times the intrinsic logs of
 	// the given header determines the prime block
+	totalDeltaSPrime := new(big.Int).Add(header.ParentDeltaS(common.REGION_CTX), header.ParentDeltaS(common.ZONE_CTX))
+	totalDeltaSPrime = new(big.Int).Add(totalDeltaSPrime, intrinsicS)
+	primeDeltaSTarget := new(big.Int).Div(params.PrimeEntropyTarget, big2)
```

Comment:
Comment would be good to explain why this line is necessary

Reply:
Have a detailed commit message explaining the rationale